apps/src/templates/courseOverview/CourseScript.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component, PropTypes } from 'react';
+import color from "@cdo/apps/util/color";
+import i18n from '@cdo/locale';
+import ProgressButton from '../progress/ProgressButton';
+
+const styles = {
+  main: {
+    width: '100%',
+    height: '100%',
+    background: color.background_gray,
+    borderWidth: 1,
+    borderColor: color.border_gray,
+    borderStyle: 'solid',
+    borderRadius: 2,
+    marginBottom: 12,
+  },
+  content: {
+    padding: 20
+  },
+  description: {
+    marginTop: 20,
+    marginBottom: 20
+  },
+  title: {
+    fontSize: 18,
+    fontFamily: '"Gotham 5r", sans-serif',
+  }
+};
+
+export default class CourseScript extends Component {
```

Comment:
Historically we weren't using classes, so this would look something like
```
const CourseScript = React.createClass({
```
In this case it's actually invalid syntax to prepend `export default`, i.e. the following will not compile:
```
export default const CourseScript = React.createClass({
```

In a world where we're using classes instead of createClass, I think either approach is fine. Often, you'll end up connecting components anyways, at which point it will usually make more sense to first define the class, and then later connect + export.